calculator/src/test/java/com/market/banica/calculator/componentTests/CalculatorComponentIT.java

```diff
@@ -274,10 +280,20 @@ public void cleanUp() {
     }
 
     @NotNull
-    private RecipeDTO createRecipeDTO() {
-        RecipeDTO response = new RecipeDTO();
+    private ProductDto createProductDTO() {
+        ProductDto response = new ProductDto();
+        ProductSpecification productSpecification = new ProductSpecification();
```

Comment:
population of ProductSpecification is not part of createProductDTO, it should be in a separate method

Reply:
Fixed